package-lock.json

```diff
@@ -32,10 +32,10 @@
       "integrity": "sha1-eHphVEFPPJntg8V3IVOyD+sM7DI=",
       "dev": true,
       "requires": {
-        "commander": "2.1.0",
-        "nan": "1.0.0",
-        "options": "0.0.6",
-        "tinycolor": "0.0.1"
+        "commander": "~2.1.0",
+        "nan": "~1.0.0",
+        "options": ">=0.0.5",
+        "tinycolor": "0.x"
       }
```

Comment:
why did this file change?

Reply:
Oops, by accident. (I'll commit a fixed package-lock.json once html-minifier and closure npm update PRs land)